memecity.game/ExpComponent.cpp

```diff
@@ -0,0 +1,17 @@
+#include "ExpComponent.h"
+
+std::string ExpComponent::COMPONENT_TYPE = "ExpComponent";
+
+ExpComponent::ExpComponent(Entity* e): Component(e)
+{
+}
+
+std::string ExpComponent::get_type()
+{
+	return std::string("ExpComponent");
```

Comment:
het idee is dat je `COMPONENT_TYPE` returned

Reply:
updated